editoast/src/modelsv2/projects.rs

```diff
@@ -0,0 +1,257 @@
+use crate::error::Result;
+use crate::modelsv2::Save;
+use crate::views::pagination::{Paginate, PaginatedResponse};
+use crate::DbPool;
+use actix_web::web::Data;
+use async_trait::async_trait;
+use chrono::{NaiveDateTime, Utc};
+use derivative::Derivative;
+use diesel::sql_types::BigInt;
+use diesel::{sql_query, ExpressionMethods, QueryDsl};
+use diesel_async::{AsyncPgConnection as PgConnection, RunQueryDsl};
+use editoast_derive::ModelV2;
+use serde::{Deserialize, Serialize};
+use utoipa::ToSchema;
+
+use crate::models::List;
+
+crate::schemas! {
+    Ordering,
+    ProjectWithStudies,
+    Project,
+    ProjectRow,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize, Derivative, ModelV2, ToSchema)]
+#[derivative(Default)]
+#[model(table = crate::tables::project)]
+#[model(row(derive(Debug, Clone, Serialize, ToSchema)))]
+pub struct Project {
+    #[schema(value_type = i64)]
+    pub id: i64,
```

Comment:
We don't need to specify if it's the same as the field type

```suggestion
    pub id: i64,
```

Reply:
Done.